### DISCUSSION TITLE
Synchronizing two raudio devices playing the same stream

### DISCUSSION THREAD

**Discussion by digitalsushi** [rern/rAudio#1698] [General]
Hi, I searched around a bit and didn't find a similar conversation, if I missed finding it then I failed in good faith.

I have two raudio rapsberry pi units on two amplifiers, each on the same ethernet lan, each have wifi disabled. I have noticed that they are several seconds out of synchronization with each other. 

I generally understand, at least metaphorically, how the streaming queue works, and that there's no expected behavior for these to be synchronized, but my obvious question is, are there any tricks for getting them within a few milliseconds of each other so that they audibly present as a single presentation?

For the sake of obviousness, the amplifiers are sufficiently difficult to share a cable between.

**Comment by AndyFx12** [comment]
Hi, 
Snapcast should do the trick. 
See snapcast server/client on settings/features.

**Reply by digitalsushi** [reply]
oh WOW. yup, that is 100% perfect. thank you
